app.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+function randomGen (min,max) {
+    return Math.floor(Math.random() * (max - min + 1) + min);
+};
+var hours = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm'];
+
+var firstAndPike = {
+  location: '1st and Pike',
+  minHourly: 23,
+  maxHourly: 65,
+  avgSale: 6.3,
+  cookiesEachHour: [],
+  custEachHour: [],
+  totalDailySales: 0,
+  calcRandCustPerHour: function () {
+    for (var i = 0; i < hours.length; i++) {
+        this.custEachHour.push(Math.ceil(Math.random() * (this.maxHourly - this.minHourly)) + this.minHourly);
+    }
+  },
+    calcTotalDailySales: function(){
+        this.calcRandCustPerHour();
+        for (var j = 0; j < hours.length; j++) {
+            this.cookiesEachHour.push(Math.ceil(this.custEachHour[j] * this.avgSale));
+            this.totalDailySales += this.cookiesEachHour[j];
+        }
+    }
+}
+
+var seatac = {
+  location: 'SeaTac Airport',
+  minHourly: 3,
+  maxHourly: 24,
+  avgSale: 1.2,
+  cookiesEachHour: [],
+  custEachHour: [],
+  totalDailySales: 0,
+  calcRandCustPerHour: function () {
+    for (var i = 0; i < hours.length; i++) {
+        this.custEachHour.push(Math.ceil(Math.random() * (this.maxHourly - this.minHourly)) + this.minHourly);
+    }
+  },
+    calcTotalDailySales: function(){
+        this.calcRandCustPerHour();
+        for (var j = 0; j < hours.length; j++) {
```

Comment:
no need to use `j` here. You're starting a new loop in a new function. Just use `i` again